contracts/BullaFinance.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.7;
+import './interfaces/IBullaClaim.sol';
+import './BullaBanker.sol';
+
+uint256 constant BPS = 10_000;
+
+/// @title An extension to BullaClaim V1 that allows creditors to finance invoices
+/// @author @colinnielsen
+/// @notice Arbitrates loan terms between a creditor and a debtor, managing payments and credit via Bulla Claims
+abstract contract BullaFinance {
+    struct FinanceTerms {
+        uint24 minDownPaymentBPS;
+        uint24 interestBPS;
+        uint40 termCompletionTimestamp;
```

Comment:
this should be termLength, not completion timestamp. Loan starts when it is accepted.

Reply:
Yeah, true. I will fix this.